### ISSUE TITLE
Scaladoc lists

### ISSUE THREAD

**Issue by sschaef** [scala-ide/scalariform#168]
_From @charpov on September 15, 2015 16:19_

I'm using lists in Scaladoc:
```
      /** Documentation
        *
        *  - item 1
        *  - item 2
        */
```
Note the __two__ spaces between `*` and `- item 1`, which are necessary to make it a list.  My problem is that scalariform reduces these two spaces to one, thus destroying the list structure.

Is there a way to deal with that?

_Copied from original issue: daniel-trinh/scalariform#78_

**Comment by sschaef** [comment]
_From @charpov on November 24, 2015 21:27_

No news on this?  It continues to mess up my code between sbt and IntelliJ IDEA.  Is there a way to turn off scalariform entirely for scaladoc comments?

**Comment by sschaef** [comment]
_From @daniel-trinh on November 30, 2015 8:0_

It looks like its possible to modify the code to not strip out extra leading whitespace. 

There's a few things that can be done:

+ Special case when the first character is the start of a scaladoc list to not strip leading whitespace
+ Add a new preference to allow no formatting of comments
+ Change the behavior in general to not strip out leading whitespace
+ Adding another preference to enable or disable trimming of whitespace in comments

The first option seems to be the "smartest", although there seem to be other reasons why someone might want to disable commenting on comments, so that might be useful to implement regardless. Not stripping out whitespace might be OK too, will need to think about this some more what the best solution is.

**Comment by jkinkead** [comment]
I'm fairly certain this was fixed in #146 . This was a bug in the handling of leading space when you enabled `PlaceScaladocAsterisksBeneathSecondAsterisk`. If @charpov wants to verify, that would be great - but closing for now; scaladoc comments have leading whitespace preserved in all cases now.
